pkg/pillar/cmd/zedagent/handlecertconfig.go

```diff
@@ -127,70 +142,181 @@ func unpublishControllerCert(ctx *getconfigContext, key string) {
 func handleEveNodeCertModify(ctxArg interface{}, key string,
 	configArg interface{}) {
 
-	// XXX TBD
+	ctx := ctxArg.(*zedagentContext)
 	status := configArg.(types.EveNodeCert)
 	log.Infof("handleEveNodeCertModify for %s", status.Key())
+	triggerEveNodeCertEvent(ctx)
 	return
 }
 
 func handleEveNodeCertDelete(ctxArg interface{}, key string,
 	configArg interface{}) {
 
-	// XXX TBD
+	ctx := ctxArg.(*zedagentContext)
 	status := configArg.(types.EveNodeCert)
 	log.Infof("handleEveNodeCertDelete for %s", status.Key())
+	triggerEveNodeCertEvent(ctx)
 	return
 }
 
 // Run a task certificate post task, on change trigger
-func certsTask(ctx *zedagentContext, triggerCerts <-chan struct{}) {
-	iteration := 0
+func controllerCertsTask(ctx *zedagentContext, triggerCerts <-chan struct{}) {
+
 	log.Infoln("starting certs publish task")
```

Comment:
Comment and log are off; this is a fetch not a publish task.

Reply:
sure